public/app.tsx

```diff
@@ -12,6 +12,7 @@ import { Navigation, APP_PATH } from './utils';
 import { Overview, UseCases, Workflows } from './pages';
 import { CoreServicesConsumer } from './core_services';
 
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
```

Comment:
For my understanding: Is this comment needed to disable eslint? If so, why are disabling it here?

Reply:
Its just to disable the next line which has an empty interface. Regularly the linting would flag this as an issue, but in this case it's expected.

common practice used in OSD which I'm following here is all